README.md

```diff
@@ -172,19 +190,43 @@ Run the following if you want to give someone else access to the deployed and ex
 
 Run `azd down`
 
-### Quickstart
+### Using the app
 
-- In Azure: navigate to the Azure Static Web App deployed by `azd`. The URL is printed out when `azd` completes (as "Endpoint"), or you can find it in the Azure portal.
+- In Azure: navigate to the Azure Contaier App deployed by `azd`. The URL is printed out when `azd` completes (as "Endpoint"), or you can find it in the Azure portal.
 - When running locally, navigate to <http://localhost:7181> for the client app and <http://localhost:7181/swagger> for the Open API server page.
 
 Once in the web app:
 
 - On the **Voice Chat** page, select the voice settings dialog and configure text-to-speech preferences.
   - You can either type messages to interact with Blazor Clippy, or select the Speak toggle button to use speech-to-text as your input.
-- Try different topics in **Chat** or **Ask Questions** context. For chat, try follow up questions, clarifications, ask to simplify or elaborate on answer, etc.
+- Try different topics in **Chat** context. For chat, try follow up questions, clarifications, ask to simplify or elaborate on answer, etc.
 - Explore citations and sources
 - Click on the "settings" icon to try different options, tweak prompts, etc.
 
+## Enabling optional features
+
+### Enabling Application Insights
+
+### Enabling authentication
+
+By default, the deployed Azure container app will have no authentication or access restrictions enabled, meaning anyone with routable network access to the container app can chat with your indexed data.  You can require authentication to your Azure Active Directory by following the [Add container app authentication](https://learn.microsoft.com/en-us/azure/container-apps/authentication-azure-active-directory) tutorial and set it up against the deployed container app.
+
+To then limit access to a specific set of users or groups, you can follow the steps from [Restrict your Azure AD app to a set of users](https://learn.microsoft.com/azure/active-directory/develop/howto-restrict-your-app-to-a-set-of-users) by changing "Assignment Required?" option under the Enterprise Application, and then assigning users/groups access.  Users not granted explicit access will receive the error message -AADSTS50105: Your administrator has configured the application <app_name> to block users unless they are specifically granted ('assigned') access to the application.-
+
+## Productionizing
```

Comment:
For the things we suggest down below, would it be better to make them parameters to Bicep (with the current as default) to allow overriding without diving into the Bicep?

Reply:
Probably

Reply:
Tracking here #197